node/podcontroller.go

```diff
@@ -256,6 +253,10 @@ func (pc *PodController) Run(ctx context.Context, podSyncWorkers int) (retErr er
 	}
 	log.G(ctx).Info("Pod cache in-sync")
 
+	provider.NotifyPods(ctx, func(pod *corev1.Pod) {
```

Comment:
Did we need to put this back to the original position?

Reply:
IMO, should not. If we put it back, the issue would still exist.